.github/workflows/issue-close-require.yml

```diff
@@ -4,6 +4,8 @@ on:
   schedule:
     - cron: "0 0 * * *"
 
+permissions: {}
```

Comment:
Since this workflow closes issues, doesn't it need the `issues: write` permission?

Reply:
Good catch 🙏